e2e/provision/init.sh

```diff
@@ -106,4 +106,6 @@ printf "%s secs\n" "$(($(date +%s) - int_start))"
 
 if [[ $RUN_E2E == "true" ]]; then
     runuser -u "$NEPHIO_USER" "$REPO_DIR/e2e/e2e.sh"
+elif [[ $RUN_E2E == "parallel" ]]; then
+    runuser -u "$NEPHIO_USER" "$REPO_DIR/e2e/e2e.sh" "true"
 fi
```

Comment:
What about another env variable?

```suggestion
if [[ $RUN_E2E == "true" ]]; then
    runuser -u "$NEPHIO_USER" "$REPO_DIR/e2e/e2e.sh" "${RUN_E2E_PARALLEL:-false}"
fi
```